pkg/operator/operator.go

```diff
@@ -108,8 +110,10 @@ func NewOperator(opts Options) Operator {
 		}
 		wd := &DaprWatchdog{
 			client:            mgrClient,
+			nonCachedClient:   mgr.GetAPIReader(),
```

Comment:
Why removing the cache? Wouldn't this increase the load on the K8s APIs?

Reply:
this is used with patch only, so the expectation is that the we hit the query once, patch, and it should not appear next time.